README.md

```diff
@@ -1,14 +1,13 @@
 # gate-resource
 
-[![Build Status](https://img.shields.io/docker/build/meshcloud/gate-resource.svg)](https://hub.docker.com/r/meshcloud/gate-resource/)
```

Comment:
there are nice GH actions build badge - https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge can you add those?